src/core/figures.js

```diff
@@ -19,7 +19,8 @@ export function run(conf) {
     if (caption) {
       decorateFigure(fig, caption, i, conf);
       figMap[fig.id] = $(caption.childNodes);
-    } else showInlineWarning(fig, "Found a `<figure>` without a `<figcaption>`");
+    } else
+      showInlineWarning(fig, "Found a `<figure>` without a `<figcaption>`");
```

Comment:
We should probably warp this "{ }"... we've not done unwrapped else before. Even though they are ok (and we have tooling to catch issues), after seeing bugs in other people's code because of unwrapped if/else statements, I'd prefer we wrap it (and yes, I know we increasingly doing unwrapped `if` nowadays in ReSpec, but the dangling else look a little odd).

Reply:
`curly` is the rule but no option fully satisfies me. I'll just add the braces manually for now.